modules/promotions/assets/js/editor/behavior.js

```diff
@@ -1,10 +1,12 @@
+import App from './e-promotion/app/app';
```

Comment:
@elchugreeva @mike-elementor 

Maybe:

```suggestion
import App from './react/app';
```

This whole module has promotions. The folder name 'e-promotion' isn't really clear to me. It is not self explanatory.